WalletWasabi.Fluent/Screenshot/Capture.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using WalletWasabi.Fluent.Screenshot.Renderers;
+
+namespace WalletWasabi.Fluent.Screenshot;
+
+public static class Capture
+{
+    public static void AttachCapture(this TopLevel root)
+    {
+        AttachCapture(root, new(Key.F6));
+    }
+
+    public static void AttachCapture(this TopLevel root, KeyGesture gesture)
+    {
+        async void Handler(object? sender, KeyEventArgs args)
+        {
+            if (gesture.Matches(args))
+            {
+                await SaveAsync(root);
+            }
+        }
+
+        root.AddHandler(InputElement.KeyDownEvent, Handler, RoutingStrategies.Tunnel);
+    }
+
+    private static async Task SaveAsync(TopLevel root)
+    {
+        var dlg = new SaveFileDialog { Title = "Save screenshot" };
+        dlg.Filters = new()
+        {
+	        new() {Name = "Svg", Extensions = {"svg"}},
+	        new() {Name = "Png", Extensions = {"png"}},
+	        new() { Name = "Pdf", Extensions = { "pdf" } },
+	        new() { Name = "Xps", Extensions = { "xps" } },
+	        new() { Name = "Skp", Extensions = { "skp" } },
+	        new() { Name = "All", Extensions = { "*" } }
```

Comment:
Nit: mind the bracket spacing consistency

Reply:
Reformatted